modules/lang-mustache/src/test/resources/rest-api-spec/test/lang_mustache/50_msearch.yaml

```diff
@@ -51,3 +51,4 @@
           - query:
               { "template": { "query": { "term": { "foo": { "value": "{{template}}" } } }, "params": { "template": "bar" } } }
   - match: { responses.0.hits.total: 1 }
+
```

Comment:
Agree, the test is messy, but so are many tests (see messy tests package). 

If me (the guy simply refactoring stuff) has to fix all of these tests, i'm going to destabilize the build: guaranteed. Thats why I simply move stuff around: the onus cannot be on me.

Alternatively, I am happy to delete the test. But i guess I am saying, I do not think it should be fixed on this issue.